_pages/software-tools/tock.md

```diff
@@ -78,11 +78,12 @@ You **must** bill for the below activities because these are direct costs. This
     - Time onboarding to a project, which includes reading documentation and learning about the agency
     - Time offboarding from a project. This includes post-mortems, writing documentation, and organizing your working files so that other people could pick it up later and continue your work
     - Business development and IAA work (for continuing projects only)
+    - Conferences and trainings that are in service of project work
```

Comment:
What do you think of "Conferences, training and meetings that are in service of project work"

Reply:
that works perfect